src/main/java/thePackmaster/orbs/runicpack/Glyph.java

```diff
@@ -0,0 +1,169 @@
+package thePackmaster.orbs.runicpack;
+
+import basemod.abstracts.CustomOrb;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.badlogic.gdx.math.Interpolation;
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.animations.VFXAction;
+import com.megacrit.cardcrawl.actions.common.DamageAction;
+import com.megacrit.cardcrawl.actions.common.GainBlockAction;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.helpers.MathHelper;
+import com.megacrit.cardcrawl.localization.OrbStrings;
+import com.megacrit.cardcrawl.orbs.AbstractOrb;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import com.megacrit.cardcrawl.powers.FocusPower;
+import com.megacrit.cardcrawl.vfx.BobEffect;
+import thePackmaster.SpireAnniversary5Mod;
+import thePackmaster.util.Wiz;
+import thePackmaster.vfx.spherespack.BlazeOrbActivateEffect;
+
+import static thePackmaster.SpireAnniversary5Mod.makePath;
+import static thePackmaster.util.Wiz.adp;
+
+public class Glyph extends CustomOrb {
+    public static final String ORB_ID = SpireAnniversary5Mod.makeID(Glyph.class.getSimpleName());
+    private static final OrbStrings orbString = CardCrawlGame.languagePack.getOrbString(ORB_ID);
+    public static final String NAME = orbString.NAME;
+    public static final String[] DESCRIPTIONS = orbString.DESCRIPTION;
+    private static final String IMG_PATH = makePath("/images/orbs/spherespack/Blaze.png");
+    private static final float SPIRIT_WIDTH = 96.0f;
+
+    private final static int BASE_PASSIVE = 1;
+    private final static int BASE_EVOKE = 4;
```

Comment:
Given that Glyphs effectively get double scaling (extra damage and extra block), what do you think about making the base evoke 3 instead of 4? So instead of getting +3 to one stat like Frost and Lightning, Glyphs get +2 to 2 stats. That still seems fair and Glyph orbs are good enough that I really don't think it would make them too weak. It would also reduce the difference between the passive effect and the evoke effect -- right now at 0 focus the evoke is 4 times the passive!